### PR TITLE
Adds textile fabricator recipes

### PR DESCRIPTION
**Pull request by Kurgis**
## Description of changes

Adds recipes for the backpack and job satchels. Adds Medical underclothing to the list, and this is of course going to need yet more work later. Changed "EMS" to "ems" because it was messing with my editor.

## Why and what will this PR improve

A bunch of recipes aren't being dumped into the default lists anymore, and allows botanical satchels to be acquired without cargo or pre-spawns.

## Authorship

Kurgis

## Changelog

:cl:
tweak: Expanded the Textiles Fabricator to have additional recipes, and removed some redundant categories.
/:cl:

### CODE REVIEW

File: code/modules/clothing/head/hardhat.dm [hunk 1]
```diff
@@ -73,7 +73,7 @@
 	icon = 'icons/clothing/head/hardhat/damage_control.dmi'
 	flags_inv = HIDEEARS|BLOCKHAIR
 
-/obj/item/clothing/head/hardhat/EMS/DC_light
```
Comment (MistakeNot4892) [inline code/modules/clothing/head/hardhat.dm:76]: We should probably put 'all types are lowercase' in the style guide.

**Review by MistakeNot4892 — APPROVED** [review]

**Comment by afterthought2** [comment]
Probably should set matter on backpacks as well, as currently:
needresources

**Comment by MistakeNot4892** [comment]
Depends on #1888

**Comment by MistakeNot4892** [comment]
Dep got merged.

**Pull request merged**